webview/platforms/winforms.py

```diff
@@ -646,6 +646,28 @@ def subprocess_args(include_stdout=True):
 _main_window_created = Event()
 _main_window_created.clear()
 
+
+class Process(object):
+    name = Name
+    alive = None
+    daemon = False
+    pid = None
+    exitcode = None
+    authkey = b''
+    sentinel = None
+    kill = None
+    join = None
+    
+    def is_alive(self):
+        return self.alive
+    
+    def terminate(self):
+        self.kill()
+        
+    def close(self):
+        self.kill()
+
+
```

Comment:
Use a common process object

Reply:
no
custom Process is used in qt and winforms
winforms api and qt api is different
but this processes are same

Reply:
Winforms and QT APIs are different but they can be unified via a common process object. This is what I am asking.